src/yahtzee_clj/core.clj

```diff
@@ -9,11 +10,80 @@
   "Rolls a hand of 5 six sided dice."
   (repeatedly 5 roll))
 
-(defn score-singles [hand]
-  "Scores the singles."
+(defn score-chance [hand]
+  "Scores the chance - i.e. sum of hand."
   (reduce + hand))
 
+(defn score-singles [hand single]
+  "Scores the singles"
+  (reduce + (filter #(= single %) hand))
+  )
+
+(defn score-of-a-kind [hand kind-type]
+  "Scores three, four and five of a kind"
+  (let [most-common (last (sort-by val (frequencies hand)))
```

Comment:
We could use some destructuring here:

[face count] (last (sort-by val (frequencies hand)))